src/mscorlib/src/System/Marvin.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This needs to be in the Shared partition. Otherwise, the reference to this won't compile it CoreRT.